airflow/www/views.py

```diff
@@ -491,6 +491,19 @@ def not_found(error):
     )
 
 
+def method_not_allowed(error):
+    """Show Method Not Allowed on screen for any error in the Webserver"""
+    return (
+        render_template(
+            'airflow/method_not_allowed.html',
```

Comment:
I feel we could merge this and the 404 template instead of duplicating most of the content.

Reply:
Agree. we could easily templatise the html template to rednder differently in  both cases.

Reply:
Thanks both. I updated the PR based on this comment, could you please take a look again?